datasources/twitter/api_test.go

```diff
@@ -0,0 +1,308 @@
+          "expanded_url": "https:\/\/www.instagram.com\/p\/BAABAABAABA\/?igshid=xyxyxyxyxyxyx",
+          "display_url": "instagram.com\/p\/BAABAABAABA\/\u2026",
+          "indices": [
+            52,
+            75
+          ]
+        }
+      ]
+    },
+    "source": "\u003ca href=\"http:\/\/instagram.com\" rel=\"nofollow\"\u003eInstagram\u003c\/a\u003e",
+    "in_reply_to_status_id": null,
+    "in_reply_to_status_id_str": null,
+    "in_reply_to_user_id": null,
+    "in_reply_to_user_id_str": null,
+    "in_reply_to_screen_name": null,
+    "geo": {
+      "type": "Point",
+      "coordinates": [
+        34.0522,
+        -118.243
+      ]
+    },
+    "coordinates": {
+      "type": "Point",
+      "coordinates": [
+        -118.243,
+        34.0522
+      ]
+    },
+    "place": {
+      "id": "3b77caf94bfc81fe",
+      "url": "https:\/\/api.twitter.com\/1.1\/geo\/id\/3b77caf94bfc81fe.json",
+      "place_type": "city",
+      "name": "Los Angeles",
+      "full_name": "Los Angeles, CA",
+      "country_code": "US",
+      "country": "USA",
+      "contained_within": [
+        
+      ],
+      "bounding_box": {
+        "type": "Polygon",
+        "coordinates": [
+          [
+            [
+              -118.668404,
+              33.704538
+            ],
+            [
+              -118.155409,
+              33.704538
+            ],
+            [
+              -118.155409,
+              34.337041
+            ],
+            [
+              -118.668404,
+              34.337041
+            ]
+          ]
+        ]
+      },
+      "attributes": {
+        
+      }
+    },
+    "contributors": null,
+    "is_quote_status": false,
+    "retweet_count": 0,
+    "favorite_count": 0,
+    "favorited": false,
+    "retweeted": false,
+    "possibly_sensitive": false,
+    "lang": "en"
+  },
+  "contributors_enabled": false,
+  "is_translator": false,
+  "is_translation_enabled": false,
+  "profile_background_color": "FFFFFF",
+  "profile_background_image_url": "http:\/\/abs.twimg.com\/images\/themes\/theme1\/bg.png",
+  "profile_background_image_url_https": "https:\/\/abs.twimg.com\/images\/themes\/theme1\/bg.png",
+  "profile_background_tile": true,
+  "profile_image_url": "http:\/\/pbs.twimg.com\/profile_images\/923335960007340032\/pIbUjNkC_normal.jpg",
+  "profile_image_url_https": "https:\/\/pbs.twimg.com\/profile_images\/923335960007340032\/pIbUjNkC_normal.jpg",
+  "profile_banner_url": "https:\/\/pbs.twimg.com\/profile_banners\/9876543\/1508975481",
+  "profile_link_color": "0012BB",
+  "profile_sidebar_border_color": "AAAAAA",
+  "profile_sidebar_fill_color": "FFFFFF",
+  "profile_text_color": "000000",
+  "profile_use_background_image": false,
+  "has_extended_profile": false,
+  "default_profile": false,
+  "default_profile_image": false,
+  "can_media_tag": null,
+  "followed_by": null,
+  "following": null,
+  "follow_request_sent": null,
+  "notifications": null,
+  "translator_type": "none"
+}
+`)
```

Comment:
I tried to anonymize the user ids etc as best as I could from the above actual API response